examples/purchaseTesterTypescript/app/screens/PaywallScreen.tsx

```diff
@@ -1,28 +1,24 @@
 import React from 'react';
-import { Paywall } from 'react-native-purchases-ui';
+import {Paywall} from 'react-native-purchases-ui';
 
-import {
-  StyleSheet,
-  View,
-} from 'react-native';
-import { NativeStackScreenProps } from "@react-navigation/native-stack";
-import RootStackParamList from "../RootStackParamList";
+import {StyleSheet, View} from 'react-native';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import RootStackParamList from '../RootStackParamList';
 
 type Props = NativeStackScreenProps<RootStackParamList, 'Paywall'>;
 
 const PaywallScreen: React.FC<Props> = () => {
+  const styles = StyleSheet.create({
+    flex1: {
+      flex: 1,
+    },
+  });
+
   return (
-    <View style={styles.container}>
-      <Paywall/>
+    <View style={styles.flex1}>
+      <Paywall style={styles.flex1} />
```

Comment:
I wonder whether we should apply this `flex1` internally to our `Paywall` or just leave it it to the developer... It feels to me like they will always have to add it so maybe we should add it internally?

Reply:
right, I am going to add it internally. No one should be setting this to anything different than taking the whole screen, in any case, even if we add it, they can override it

Reply:
Done in c6adceb64f60744f6fae41b5410ae58da44ed231